src/models/__tests__/helpers/mockCompanyHouse.ts

```diff
@@ -1,14 +1,14 @@
 import { container } from 'tsyringe'
 import { Dispatcher, MockAgent, getGlobalDispatcher, setGlobalDispatcher } from 'undici'
-import { Env } from '../../../env'
+import { Env } from '../../../env.js'
 import {
   invalidCompanyNumber,
   noCompanyNumber,
   successResponse,
   validCompanyNumber,
-} from '../fixtures/companyHouseFixtures'
+} from '../fixtures/companyHouseFixtures.js'
 
-const env = container.resolve(Env)
```

Comment:
definitely shouldn't be using `any` here. If you need a type annotation you should be able to use `Env`

Reply:
ooops:) agreed and it has been updated to be itself.